### ISSUE TITLE
Community Feedback for Online Workshops

### ISSUE THREAD

**Issue by maneesha** [carpentries/conversations#28]
We would like to announce that Online Workshops will officially be moving out of Pilot and become a permanent offering in 2022. The Core Team is currently planning and identifying all the necessary steps to make it easier for everyone to host/teach/coordinate online workshops. 

We would like to hear from you the following:
- What are the advantages of hosting/teaching an online workshop? 
- What are disadvantages of hosting/teaching an online workshop? 
- What additional information would you like to share regarding online workshops and what the Core Team can do to make them successful? 

(edited for formatting)

**Comment by kariljordan** [comment]
Thanks @maneesha! How long will this issue be open?

**Comment by sheraaronhurt** [comment]
@kariljordan this will be open for the remainder of the year. The blogpost that went out last week includes this issue and I will have it in the next 2 newsletters and will be mentioned during the community call next week.

**Comment by Stonepeople** [comment]
OK - I will bite!
As well as assisting on a couple of online Carpentries workshops, I do a lot of online instructing on tech/journalism for other clients. My few centsworth - 
**What are the advantages of hosting/teaching an online workshop?**
- the fact that code and the screen are front and centre of so much of the sessions make online a great way to teach code: participants can see in detail what's happening on the screen, and follow along more easily than squinting at a big screen in a physical classroom
- the fact that there's next to no travel involved for anyone instructing or attending make it much more flexible in terms of delivery
**What are disadvantages of hosting/teaching an online workshop?**
- The intensity - the fact that one is likely to feel isolated as instructor or participant, and the dominance of the screen on one's computer make it more intense than a classroom event
- As a result the passage of time feels different - a day on zoom feels much longer than a day in a classroom, partly because without physical cues, one can easily do a long chunk of live coding, with participants simply copying what the instructor is doing. Is there really much transfer of knowledge going on here? 
**What additional information would you like to share regarding online workshops and what the Core Team can do to make them successful?**
- In converting to online delivery with one of my clients we had time to experiment at the beginning of the pandemic. We quickly realised that 2 hour sessions each day for 4-5 days would work much better than 1-2 whole days. We broke everything up into shorter modules and give participants tasks to do before the next day's session. We also put videos and supporting documentation online for them to access between sessions. In the calls for instructors I see a lot of organisers still planning  whole day and half day sessions. I am guessing that I may not be alone in feeling that devoting a day to a Carpentries workshop online is harder to do than devoting a day in a classroom used to be. Maybe we should be advising organisers that several short sessions are better than fewer long ones. In my experience in non-Carpentries workshops we get better engagement and more participants signing up than we did with longer in-person workshops. This is a "zoom-dividend" we could be looking to exploit, especially since online is here to stay!

**Comment by marklcrowe** [comment]
My thoughts/experiences

Positive

- Accessibility - by far the biggest benefit for us. We've been able to reach far more participants from right across the state who would never had the chance to attend an in-person workshop (at small universities/remote campuses without the numbers or instructors to run a course). Also much easier for students with mobility issues, carer demands, all sorts of other barriers to in-person attendance
- Flexibility - without the need to book rooms/travel to venues, it's possible to have a much more flexible range of workshop formats. Mostly we now do four consecutive half-days, but have used multiple other formats.
- Instructor feedback - Online workshops gives the opportunity to work with a much wider range of other instructors, and the more people you work with, the more new training techniques and approaches you are exposed to and can learn from (and the more diverse feedback you can receive).
- Direct code sharing - Zoom chat, Etherpad, Google Docs are all used widely and you can share code snippets in them. While we're not about cut and paste, sometimes being able to quickly duplicate code between machines can help people catch up, or the instructor more easily duplicate and check for typos, etc.

Negative

- Speed - it seems to take much longer to get though content online than I remember it doing so in person, and we find it's simply not feasible to do the full SWC lesson program in two days - we nearly always do just the R or Python component and still sometimes struggle to finish that.
- Engagement - there's no substitute for being able to identify learners who need extra support from body language, or helpers walking around the room asking people how they're getting on. There's a quiet cohort who you barely hear from in an online class and never can be sure if they're quietly happy or completely lost. Especially those who leave their cameras turned off.
- Isolation - not just learning on your own, but in providing support. As soon as you move to a breakout room to help someone, you loose all connection with the main lesson. In a classroom, you can keep some awareness of what's happening around you.
- Dropouts - not initial attendance (we've got that kind of under control) but the lack of commitment by many learners. Few people would think of walking out of an in-person class to attend another meeting for an hour, but in the Zoom world, there seems to be a mindset that you can just drop in and out of a workshop as you like. Then people come back and either are lost for much of the following material because they've missed critical content, or are unreasonably demanding on helpers (unreasonable because it's their own choice that they are behind, and their demands impact on others in the class).
- Zoom fatigue - seems to be starting to have an effect on registration numbers, especially now other activities are returning to in-person. People have just had enough of screens for now.

Neutral

- Technical issues - we were initially concerned about bandwidth and other technical challenges. But they don't seem to have been much worse or more frequent than I've experienced in-person (fire alarms, power failures, no power sockets, projector/screen failures, double-booked rooms - you name it!)

I think those are my main pros and cons. tl;dr version: Good - accessibility, flexibility. Bad - loss of engagement/connection with learners.

**Comment by rpep** [comment]
Pros:
* Had candidates who were for e.g. seconded to other organisations, based in other cities, etc. who wouldn't have attended an in-person workshop.
* Always a struggle to book rooms on campus large enough, obviously this not a problem.

Negatives:
* Lower level of attendance. Many more people would sign up then drop out before the class started.
* People leaving part way through for e.g. to go to a meeting is really common. People don't tend to do that in person, seem to be less embarassed to do it on Zoom.
* Difficulty tackling technical issues. People would sometimes drop off of the course if they couldn't follow rather than seek help. Alternatively we tried placing into breakout rooms, but then people fell behind significantly. 
* 2 days over Zoom is way too long. We taught individual lessons rather than whole curriculum for practical reasons.
* Lack of connection, often feels like talking to an empty room. Tried various things to get engagement but most people want to leave cameras off and not talk over Zoom.
